CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 cmake_minimum_required(VERSION 3.3)
 project(enrico Fortran C CXX)
 
+option(NEK_BUILD  "Determines if Nek5000 will be included" ON)
```

Comment:
Could you change this to `USE_NEK`?  That way, it will match the the `USE_SHIFT` variable from #97